test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Queues/QueueTests.cs

```diff
@@ -175,6 +175,60 @@ public void InvokeWithBindingData()
             Assert.Equal("123", msgs[0].AsString);
         }
 
+        public class ProgramWithTriggerAndCompoundBindingData
+        {
+            public class Poco
+            {
+                public SubOject prop1 { get; set; }
+                public string xyz { get; set; }
+            }
+
+            public class SubOject
+            {
+                public string xyz { get; set; }
+            }
+
+
+            // BindingData is case insensitive. 
+            // And queue name is normalized to lowercase. 
+            public const string QueueOutName = "qName-{prop1.xyz}";
+            public void Func([QueueTrigger(QueueName)] Poco triggers, [Queue(QueueOutName)] ICollector<string> q)
```

Comment:
Can you also add a `string prop1` parameter to this method? I.e., hopefully these binding expressions can be bound to as normal parameters? In this case, I think we might already have poco handling which will take care of this - my question is more for properties that are resolved from metadata other than from the poco - e.g. CloudBlockBlob.Metadata which the blob trigger adds. Can we dot into the metadata as a parameter?